.github/workflows/commit_lint.yml

```diff
@@ -10,4 +10,4 @@ jobs:
           fetch-depth: 0
       - uses: wagoid/commitlint-github-action@v5
         with:
-          commitDepth: 1
\ No newline at end of file
+          commitDepth: -1
```

Comment:
What is the purpose of this ?

Reply:
From me man =))))))), it's for check latest commit only @mthinh , not all commit in this PR